Makefile

```diff
@@ -192,10 +187,6 @@ protos: gotools
 	@echo "Compiling non-API protos..."
 	./scripts/compile_protos.sh
 
-.PHONY: changelog
```

Comment:
How are we generating the change log now?

Reply:
I've actually never used the make target for changelog or the PREV_VERSION variable... I've always just run the script directly, passing in the old and new tag, e.g.
`./scripts/changelog.sh v2.0.0 v2.0.1`